docs/unit-converter-requirements.md

```diff
@@ -0,0 +1,17 @@
+# Make a unit converter
+
+## Interface
+Get creative.
```

Comment:
Ah, so between your two inline responses, I think we can keep the title as is but maybe structure the content in this interface section to be more similar to your comment here. "Design the interface however you like. If you're feeling stuck, try to recreate..." (feel free to re-word but something similar to that gets your point across)